core/handler_schema_constants.py

```diff
@@ -35,7 +35,6 @@
     'BulkEmailWebhookEndpoint',
     'DeferredTasksHandler',
     'DeleteAccountPage',
-    'EditableQuestionDataHandler',
```

Comment:
You need to remove more handlers from this file.

Reply:
i have removed all handlers i'm working on
which handler should i remove again ?

Reply:
You are removing `EditableQuestionDataHandler` and `QuestionCountDataHandler`, what about `EditableSkillDataHandler`?

Reply:
![image](https://user-images.githubusercontent.com/48652493/154563507-35e6ab25-a580-44ad-85b8-34447946c9f8.jpeg)

That handler is not available in the `handler_schema_constants.py` file looks like somebody else is working on it

Reply:
`EditableSkillDataHandler` is already among the list of completed handler classes.

What should i do? I didn't know anyone else was working on it?